package.json

```diff
@@ -47,11 +47,14 @@
     "@walletconnect/client": "2.0.0-beta.9",
     "@walletconnect/types": "2.0.0-beta.9",
     "@walletconnect/utils": "2.0.0-beta.9",
+    "@types/papaparse": "^5.2.2",
     "axios": "^0.21.1",
     "better-sqlite3": "^7.1.2",
     "bignumber.js": "^9.0.1",
     "electron-log": "^4.3.1",
     "pino": "^6.11.3",
+    "ethers": "^5.0.31",
```

Comment:
is this dependency necessary? BigNumber already exists as direct dep and similar util functions already exist in various web3 utils packages that the app already depends on.

Since contractkit and rest of the app all depend on web3 we don't really want to add dependency on ethers unless it is absolutely necessary.

Reply:
I already fixed